tasks/archive/2026/03/20260302-charts-phase1-todo.md

```
@@ -0,0 +1,1574 @@
+  colorPalette?: string;
+};
+```
+
+**Step 3: Run type check**
+
+Run: `cd packages/frontend && pnpm tsc --noEmit`
+Expected: PASS (all new fields are optional, backward compatible)
+
+**Step 4: Commit**
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)